openpathsampling/pathmover.py

```diff
@@ -802,10 +812,9 @@ def __call__(self, sample1, sample2):
 # SUBTRAJECTORY GENERATORS
 ###############################################################################
 
-
-class RandomSubtrajectorySelectMover(SampleMover):
+class SubtrajectorySelectMover(SampleMover):
```

Comment:
I also like the separation of the abstract `SubtrajectorySelect` from `RandomSubtrajSelect`